src/EditorFeatures/CSharpTest/Completion/CompletionProviders/KeywordCompletionProviderTests.cs

```diff
@@ -357,7 +357,6 @@ class C
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
         public async Task PrivateProtectedModifier()
         {
-            System.Diagnostics.Debugger.Break();
```

Comment:
@rchande Do you know why these were here?

Reply:
it is probably fixed here https://github.com/dotnet/roslyn/pull/22158

Reply:
@jcouv fixed that.